lib/doc/worksheet.js

```diff
@@ -344,6 +344,49 @@ class Worksheet {
     });
   }
 
+  duplicateRow(start, count) {
+    // I want to add after the start row
+    start++;
+
+    const nKeep = start;
+    const nEnd = this._rows.length;
+    let i;
+    let rSrc;
+
+    // insert new cells
```

Comment:
Thinking out loud here - this for loop is a duplication of code already implemented in worksheet.spliceRows()
Perhaps a better strategy would be to prepare an array of insertion values and let spliceRows() do the donkey work...
```javascript
const values = this._rows[start - 1].values;
const inserts = new Array(count - 1).fill(values);
const deleteCount = insert ? 0 : count - 1;
this.spliceRows(start + 1, deleteCount, ...inserts);

// copy styles from start row to cloned rows
...
```